.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "baselines/esm"]
+	path = baselines/esm
+	url = git@github.com:facebookresearch/esm.git
```

Comment:
I'd pin the specific version/commit, given that things may change ;)

Reply:
good point, I just added a version tag to this